client/components/editor/editor-markdown.vue

```diff
@@ -454,21 +455,36 @@ export default {
       this.processContent(newContent)
     }, 600),
     onCmPaste (cm, ev) {
-      // const clipItems = (ev.clipboardData || ev.originalEvent.clipboardData).items
-      // for (let clipItem of clipItems) {
-      //   if (_.startsWith(clipItem.type, 'image/')) {
-      //     const file = clipItem.getAsFile()
-      //     const reader = new FileReader()
-      //     reader.onload = evt => {
-      //       this.$store.commit(`loadingStart`, 'editor-paste-image')
-      //       this.insertAfter({
-      //         content: `![${file.name}](${evt.target.result})`,
-      //         newLine: true
-      //       })
-      //     }
-      //     reader.readAsDataURL(file)
-      //   }
-      // }
+      const clipItems = (ev.clipboardData || ev.originalEvent.clipboardData).items
+      for (let clipItem of clipItems) {
+        if (_.startsWith(clipItem.type, 'image/')) {
+          const file = clipItem.getAsFile()
+
+          const now = new Date()
+          // file name format yyyy-MM-dd_random.png
+          let filename = `${now.getFullYear()}-${now.getMonth()+1}-${now.getDate()}_${parseInt(Math.random() * (10**5))}`
```

Comment:
under which folder will this image be stored?

Reply:
in the root directory
```js
          const form = new FormData()
          form.append('mediaUpload', JSON.stringify({ 'folderId':0 }))
          form.append('mediaUpload', file, filename)
```
[L469](https://github.com/requarks/wiki/pull/5441/commits/4f7e08e6ccbe297ce52fb37d3c3b35334d250c28#diff-6572222f98358241abe369388929bd7692fc070e1ef397ec605ee31f83f984d0R469)

Reply:
i see, thanks. Btw how can i open issues on this github project now? i cant find the issue tracker anymore

Reply:
Admin closed it. Maybe you should go to the https://github.com/requarks/wiki/discussions

Reply:
mm does not look good to me, i hope version 3 will be eventually released and that the project will not be abandoned. Will probably ask about the roadmap in the discussion.

thanks for your answers

Reply:
> in the root directory
> 
> ```js
>           const form = new FormData()
>           form.append('mediaUpload', JSON.stringify({ 'folderId':0 }))
>           form.append('mediaUpload', file, filename)
> ```
> 
> [L469](https://github.com/requarks/wiki/pull/5441/commits/4f7e08e6ccbe297ce52fb37d3c3b35334d250c28#diff-6572222f98358241abe369388929bd7692fc070e1ef397ec605ee31f83f984d0R469)

The folder ID should be the folder of the page where you paste the img.
With this you will avoid permission issues.